presto-docs/src/main/sphinx/release/release-0.260.rst

```diff
@@ -0,0 +1,36 @@
+=============
+Release 0.260
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a bug in sql functions that would cause compiler error when session property ``inline_sql_function`` is set to ``false`` and sql function references input in lambda expression.
+* Fix multithread writing for fragment result cache.
```

Comment:
This is too low level. We should write it in a way such that it is clear what is the end impact. Did this lead to a query failure ? Did this lead to a performance regression?

Reply:
@shixuan-fan Can you provide more details here since you reviewed this code?

Reply:
How about: `Fix a bug in fragment result cache that might cause query failure`?

Reply:
`Fix a bug in fragment result cache that caused query failures`